packages/cfd/src/Containers/jurisdiction-modal/__test__/jurisdiction-card.spec.tsx

```diff
@@ -0,0 +1,254 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import JurisdictionCard from '../jurisdiction-card';
+import { Jurisdiction } from '@deriv/shared';
+
+describe('JurisdictionCard', () => {
+    type TMockProps = {
+        account_status: {
+            authentication: {
+                document: {
+                    status: 'none' | 'pending' | 'expired' | 'verified' | 'rejected';
+                };
+                identity: {
+                    services: {
+                        idv: {
+                            status: 'none' | 'pending' | 'expired' | 'verified' | 'rejected';
+                        };
+                        onfido: {
+                            status: 'none' | 'pending' | 'expired' | 'verified' | 'rejected';
+                        };
+                        manual: {
+                            status: 'none' | 'pending' | 'expired' | 'verified' | 'rejected';
+                        };
+                    };
+                };
+                needs_verification: string[];
+            };
+            currency_config: {
+                [k: string]: { is_deposit_suspended?: 0 | 1 | undefined; is_withdrawal_suspended?: 0 | 1 | undefined };
+            };
+            p2p_status: 'none' | 'active' | 'temp_ban' | 'perm_ban';
+            prompt_client_to_authenticate: 0 | 1;
+            risk_classification: string;
+            status: string[];
+        };
+        account_type: 'financial' | 'synthetic';
+        disabled: boolean;
+        jurisdiction_selected_shortcode: string;
+        setJurisdictionSelectedShortcode: jest.Mock;
+        type_of_card: 'svg' | 'bvi' | 'labuan' | 'maltainvest' | 'vanuatu';
+    };
+
+    let mock_props: TMockProps = {
```

Comment:
```suggestion
    let mock_props: TJurisdictionCardProps = {};
```
no need to use TMockProps, and no need to declare the entire object here because it will be overridden right away by mock_props = {...} on every beforeEach() run🙂

Reply:
@shaheer-deriv is it resolved?